src/workers/transactions/userReceivedAward.ts

```diff
@@ -59,6 +60,24 @@ export const userReceivedAward =
         const sender = await transaction.sender;
         const receiver = await transaction.receiver;
 
+        const [userPost, userComment] = await Promise.all([
+          queryRunner.manager.getRepository(UserPost).findOneBy({
+            awardTransactionId: transaction.id,
+          }),
+          queryRunner.manager.getRepository(UserComment).findOneBy({
+            awardTransactionId: transaction.id,
+          }),
+        ]);
+
+        let targetUrl = `/${receiver.username}`;
+
+        if (userPost) {
+          targetUrl = `/posts/${userPost.postId}`;
```

Comment:
is `COMMENTS_PREFIX` added somewhere inside notification context, not sure?

Reply:
No, probably need it for extension 😬

Reply:
Added